modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala

```diff
@@ -522,26 +522,30 @@ trait RunScriptTestDefinitions { _: RunTestDefinitions =>
   test("script wrappers satisfy strict compiler flags") {
     val inputs = TestInputs(
       os.rel / "strictClassWrapper.sc" ->
-      """//> using scala 3.3.1
```

Comment:
I am fine with the formatting changes, but curious why this happend.

Reply:
I merely ran `./mill -i __.fix && scalafmt`
curious indeed

Reply:
CI format checks suceeded, so I guess it was unformatted before? how come 🤔

Reply:
Some looking into our format checks is required, this is not the first time it happens :(